File: 1061.cpp
#include<iostream>
#include<stdio.h>
using namespace std;
int main()
{
   int sd, ed, sh, sm, ss, eh, em, es, fd, fh,fm, fs;
   string a;
   cin>>a>>sd;
   cin>>sh>>a>>sm>>a>>ss;
   cin>>a>>ed;
   cin>>eh>>a>>em>>a>>es;

   long d= ((ed*86400)+(eh*3600)+(em*60)+es)-((sd*86400)+(sh*3600)+(sm*60)+ss);
   fd= d/86400;
   fh= (d%86400)/3600;
   fm= ((d%86400)%3600)/60;
   fs= (((d%86400)%3600)%60);

   cout<<fd<<" dia(s)"<<endl;
   cout<<fh<<" hora(s)"<<endl;
   cout<<fm<<" minuto(s)"<<endl;
   cout<<fs<<" segundo(s)"<<endl;

    return 0;
}

File: 1061.py
a, sd = input().split()
sd = int(sd)
sh, sm, ss = map(int,input().split(":"))

a, ed = input().split()
ed = int(ed)
eh, em, es = map(int,input().split(":"))

d = ((ed*86400)+(eh*3600)+(em*60)+es)-((sd*86400)+(sh*3600)+(sm*60)+ss)
fd = int(d/86400)
fh = int((d%86400)/3600)
fm = int(((d%86400)%3600)/60)
fs = int((((d%86400)%3600)%60))

print(f"{fd} dia(s)")
print(f"{fh} hora(s)")
print(f"{fm} minuto(s)")
print(f"{fs} segundo(s)")

